.github/pull_request_template.md

```diff
@@ -1,28 +1,40 @@
-### Summary
+## About this template
 
-<!-- Provide a general description of the code changes in your pull
-request... were there any bugs you had fixed? If so, mention them. If
-these bugs have open GitHub issues, be sure to tag them here as well,
-to keep the conversation linked together. -->
+This template aims to help contributors to write better descriptions.
+Feel free to discard it if you need to.
```

Comment:
I think this really needs to be in a comment block too... if not, it will get left untouched often, and every PR notification will end up starting with the same words.

Reply:
@eileencodes @matthewd I put them into a comment. I also added some more comments for motivation section.